sdk/appconfiguration/azure-data-appconfiguration/src/main/java/module-info.java

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.data.appconfiguration {
+    requires com.azure.core;
+
```

Comment:
This should `exports com.azure.data.appconfiguration` and `exports com.azure.data.appconfiguration.models`

Reply:
Good point! Updated